ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -26,26 +18,10 @@
     REST_FRAMEWORK = {}
 
 
-# This is needed for the rest_filters app, but someone may use the filter class
-# without enabling the ansible_base.rest_filters app explicitly
-# we also apply this to views from other apps so we should always define it
-try:
-    ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES
-except NameError:
-    ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES = (
-        'page',
-        'page_size',
-        'format',
-        'order',
-        'order_by',
-        'search',
-        'type',
-        'host_filter',
-        'count_disabled',
-        'no_truncate',
-        'limit',
-        'validate',
-    )
+for key, value in vars(_dab_default_settings.general).items():
+    if key in locals():
+        continue
+    locals()[key] = value
```

Comment:
FWIW, it is not suggested to do this

https://docs.python.org/3/library/functions.html#locals

> Note The contents of this dictionary should not be modified; changes may not affect the values of local and free variables used by the interpreter.